modules/reindex/src/internalClusterTest/java/org/opensearch/index/codec/MultiCodecReindexIT.java

```diff
@@ -135,7 +135,7 @@ private void assertReindexingWithMultipleCodecs(String destCodec, String destCod
     }
 
     private void useCodec(String index, String codec) throws ExecutionException, InterruptedException {
-        assertAcked(client().admin().indices().prepareClose(index));
+        assertAcked(client().admin().indices().prepareClose(index).setWaitForActiveShards(1));
```

Comment:
Would `ActiveShardCount.DEFAULT` or `ActiveShardCount.ALL` be better here?

Reply:
I'm going to merge this to fix the flaky test ASAP, but we can follow up here if necessary.